package.json

```diff
@@ -26,11 +26,18 @@
     "@commitlint/config-conventional": "^17.4.3",
     "@commitlint/prompt-cli": "^17.4.3",
     "@lokalise/eslint-config-lokalise-frontend": "0.3.0",
+    "@semantic-release/changelog": "^6.0.2",
```

Comment:
🟢 I had hoped to keep this project's dependencies pinned to exact versions rather than `^` versions. Perhaps something to discuss as a team?

Reply:
If we do decide to stick to exact versions, happy for that to be part of a future task/ticket, so not a blocker for this PR.

Reply:
Adjusted, @artursvonda ok with you as well?